synapse/tests/test_lib_types.py

```diff
@@ -1,14 +1,67 @@
 # -*- coding: UTF-8 -*-
 import base64
 
-import synapse.cores.common as s_cores_common
 
 import synapse.common as s_common
 import synapse.lib.types as s_types
+import synapse.lib.xact as s_xact
 
 from synapse.tests.common import *
 
-class DataTypesTest(SynTest):
+
+class TestTypes(SynTest):
+
+    def test_hex_type(self):
+        with self.getTestCore() as core:
+            # fixme - getTestCore should have this dude loaded in him already!
```

Comment:
Should we fix this before merging?

Reply:
no - we should do that when we unify all the stuff from `iq` back into tests.

Reply:
OK makes sense